Week1/solution 3.py

```diff
@@ -0,0 +1,12 @@
+
+a=int(input("Enter number:"))
+t=a
+rev=0
+while(a>0):
+    dig=a%10
+    rev=rev*10+dig
+    a=a//10
+if(t==rev):

```

Comment:
what is the meaning of if line give me answer.

Reply:
there will be give the condition